pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -573,6 +579,7 @@ private PGStream enableSSL(PGStream pgStream, SslMode sslMode, Properties info,
       // Allow ==> start with plaintext, use encryption if required by server
       return pgStream;
     }
+    SslNegotiation sslNegotiation = SslNegotiation.of(Nullness.castNonNull(PGProperty.SSL_NEGOTIATION.getOrDefault(info)));
 
     LOGGER.log(Level.FINEST, " FE=> SSLRequest");
```

Comment:
WDYT of updating the log to accommodate for `SSLRequest(sslnegotiation=direct)` vs `SSLRequest(sslnegotiation=postgres)`?